### ISSUE TITLE
Swipe between floorplans 

### ISSUE THREAD

**Issue by cbra1** [domoticz/domoticz#4017]
Today one can only flip through the floorplans by the bullets at the bottom. Especially for mobile devices it would enhance the usability by additionally enabling **horizontal swiping** between the plans.

**Comment by dnpwwo** [comment]
Horizontal swiping has always been enabled on Apple devices.  What are you using?

**Comment by cbra1** [comment]
I use several Android devices with Chrome browsers:

Smartphone: Android 9 (pure stock version because of Android one) on a Xiaomi Mi A1

Tablet: Android 8.1 on a Lenovo Tab4 10

**Comment by frnandu** [comment]
+1

**Comment by rezzalex** [comment]
+1

**Comment by gizmocuz** [comment]
This could probably be done with HamerJS

https://hammerjs.github.io/

If someone would like to make a PR to implement this, that would be great!

**Comment by cbra1** [comment]
Yes, please, please!

**Comment by gizmocuz** [comment]
@cbra1 , is this already possible? seems like an old issue

**Comment by cbra1** [comment]
No, it is still not possible.

Yes, it is an old issue. But nobody with respective skills seemed to have heard your plea to take action 😉

However: Would be really nice to have this feature. But if you want to close this issue to get a better overview of the really important things, then that would be perfectly fine with me.

**Comment by gizmocuz** [comment]
I thought you had like push buttons on the left/right side of the floorplan?

I will have a look at it.. (not using the floorplan myself)

**Comment by cbra1** [comment]
Exactly. Little bullets at the bottom that have to be touched or clicked.

<img width="864" height="1872" alt="Image" src="https://github.com/user-attachments/assets/d1f7450a-2546-45c0-a974-413b8402a31b" />

**Comment by gizmocuz** [comment]
Fixed in commit 1fdd95984. The existing touch handlers were relying on `window.pageXOffset` to detect which floorplan to snap to, but since `body` has `overflow:hidden` the page never actually scrolls horizontally, so `pageXOffset` was always 0 and every swipe snapped back to the first floorplan.

The fix records the touch start coordinates in `touchstart` and computes a horizontal delta in `touchend`. A swipe of more than 50px that is more horizontal than vertical navigates to the previous or next floorplan. Double-tap to toggle fullscreen is preserved.

Tested between two floorplans on a touch device. Works on Android Chrome and iOS/iPadOS Safari.

**Comment by gizmocuz** [comment]
Implemented in beta version 17944

**Comment by cbra1** [comment]
OMG - it works! What a pitty that you have to do 99% yourself. Thank you!

**Comment by gizmocuz** [comment]
Got some help from Claude code as well... 👌
